cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/servlet/ServletRequestContext.java

```diff
@@ -158,6 +159,14 @@ public Map<String, Object> getParameterConfigOverrides() {
         return overrides;
     }
 
+    @Override
+    public String getUserPrincipal() {
+        Principal userPrincipal = _request.getUserPrincipal();
+        return userPrincipal == null
+                ? ""
```

Comment:
A string like "null" or "none" may be more informative.

Reply:
Ack